x/staking/types/validator.go

```diff
@@ -400,14 +401,30 @@ func (v Validator) DelegatorShareExRate() sdk.Dec {
 	return sdk.NewDecFromInt(v.Tokens).Quo(v.DelegatorShares)
 }
 
-// Get the bonded tokens which the validator holds
+// get the bonded tokens which the validator holds
 func (v Validator) BondedTokens() sdk.Int {
 	if v.Status == sdk.Bonded {
 		return v.Tokens
 	}
 	return sdk.ZeroInt()
 }
 
+var powerReduction = new(big.Int).Exp(big.NewInt(10), big.NewInt(9), nil)
+
+// get the Tendermint Power
+// a reduction of 10^9 from validator tokens is applied
+func (v Validator) Power() int64 {
+	if v.Status == sdk.Bonded {
```

Comment:
well we could reference _both_ `BondedTokens` and `PotentialPower` but I think this would be a more roundabout way to do this as we'd be duplicating all the `PotentialPower` calculations in `Power` - splitting hairs here, but I think it makes the most sense to leave as is